statsmodels/duration/hazard_regression.py

```diff
@@ -446,7 +442,7 @@ def fit(self, groups=None, **args):
 
         return results
 
-
+    
```

Comment:
You can configure your editor to strip trailing whitespace on save.  We want to avoid the history being affected by this.